tests/main/interfaces-mount-control/task.yaml

```diff
@@ -0,0 +1,52 @@
+summary: Test for the mount-control interface
+
+environment:
+    MOUNT_SRC: /var/tmp/test-snapd-mount-control
+    SNAP_COMMON: /var/snap/test-snapd-mount-control/common
+
+prepare: |
+    mkdir -p "$MOUNT_SRC/dir1"
+    echo "Something" > "$MOUNT_SRC/file1"
+
+restore: |
+    rm -rf "$MOUNT_SRC"
+
+execute: |
+    SNAP_NAME="test-snapd-mount-control"
+    MOUNT_DEST="$SNAP_COMMON/target"
+
+    echo "Installing the test snap"
+
+    "$TESTSTOOLS"/snaps-state install-local "${SNAP_NAME}"
+    if os.query is-trusty; then
+        snap warnings | tr -d '\n' | sed -e 's/\s\+/ /g' | MATCH "systemd version 204 is too old"
+        exit 0
+    fi
+
+    echo "Connecting the mount-control interface"
```

Comment:
Can a spread test be added with a snap.yaml that contains invalid `what/where/type/options` to check these get validated and the interface fails to connect etc?

Reply:
I think these are well tested in the unit tests, maybe I can add one case to the spread tests as well, but given the weight of spread tests they are probably not the best place to test all the possible cases.

Reply:
I added one test.

Reply:
Thanks.